rust-src/id/src/types.rs

```diff
@@ -801,7 +802,7 @@ pub struct ChoiceArParameters {
 /// Proof that the data sent to the identity provider
 /// is well-formed.
 #[derive(Serialize)]
-pub struct PreIdentityProof<P: Pairing, C: Curve<Scalar = P::ScalarField>> {
+pub struct PreIdentityProofOld<P: Pairing, C: Curve<Scalar = P::ScalarField>> {
```

Comment:
Is this meant to be removed after you've generated the relevant test data?

Reply:
Yes.